app/src/main/java/com/babylon/wallet/android/MainActivity.kt

```diff
@@ -61,7 +61,10 @@ class MainActivity : FragmentActivity() {
         super.onCreate(savedInstanceState)
         cloudBackupSyncExecutor.startPeriodicChecks(lifecycleOwner = this)
 
-        intent.data?.let { viewModel.handleDeepLink(it) }
+        intent.data?.let {
+            intent.replaceExtras(Bundle())
```

Comment:
Very nice that you added this here.

Reply:
actually can someone explain me what does this line do? Because I see it in the line 103, too.
Does it "clear" the intent with an empty bundle?

Reply:
it removes `data` from an intent, so deep link is not delivered 2nd time if you move app to background and open from recent